src/__experimental__/components/decimal/decimal.component.js

```diff
@@ -34,45 +31,23 @@ class Decimal extends React.Component {
       "Decide between using a controlled or uncontrolled input element for the lifetime of the component";
     const isControlled = this.isControlled();
     invariant(this.state.isControlled === isControlled, message);
-
+    if (prevProps.precision !== this.props.precision) {
+      // eslint-disable-next-line no-console
+      console.error(
+        "Decimal `precision` prop has changed value. Doing this will affect the value displayed to the user."
```

Comment:
This isn't true any more. 

```suggestion
        Changing the "Decimal `precision` prop has no effect.
```

Reply:
Changed this error message.